Specs/DateTime/Japanese/DateParser.json

```diff
@@ -2185,12 +2185,12 @@
         "Text": "今月の27日",
         "Type": "date",
         "Value": {
-          "Timex": "XXXX-11-27",
+          "Timex": "2016-11-27",
```

Comment:
Similar to other timex question. If English is the same, please add a Comment attribute to the relevant specs with this notice.

Reply:
comment added